files/fr/web/api/css_counter_styles/index.html

```diff
@@ -0,0 +1,31 @@
+---
+title: Module CSS Styles de compteurs
+slug: Web/API/CSS_Counter_Styles
+translation_of: 'Web/API/CSS_Counter_Styles'
+---
+<div>{{DefaultAPISidebar("CSS Counter Styles")}}{{SeeCompatTable}}</div>
+
+<p>Le module CSS Styles de compteurs permet de définir des styles de compteurs personnalisés qui peuvent être utilisés pour les marqueurs de listes CSS et les compteurs des contenus générés.</p>
```

Comment:
As this is inline, I would definitely add the English version in parenthesis here.

Reply:
Agreed. New proposal in https://github.com/mdn/translated-content/pull/1766/commits/b819ee39bcf12645dec134db59438b571dbf2f48